### DISCUSSION TITLE
⛔ Adjust "Upload SVG" action label

### DISCUSSION THREAD

**Discussion by arcaswebdesign** [elementor/elementor#20107] [Feature Request]
Any time there is an icon selector control (for example in the Icon List widget), the box has two buttons to click on, one for **Icon Library** and one for **Upload SVG**.

For usability reasons I would suggest a change to the language of the second one to be something like **Upload/Select SVG** as we will not always be uploading a new SVG but often selecting an existing one and the action already allows us to do both.

**Comment by nicholaszein** [comment]
Hello, @arcaswebdesign!

Thank you for submitting your suggestion! 🙏 

### ⛔ 🚀 After reviewing your request, we have decided not to add this to our roadmap for the time being.

This change does not align with other naming conventions we have and also increases the text size for other languages. When you click on the button, it shows you the familiar Media Library interface, which already shows all SVG images in the library in case you want to select one.

Please feel free to contact us again with other feature suggestions. 😉 

Best regards
